src/chrome/content/rules/Casual-Effects.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Casual-Effects.com">
+	<target host="casual-effects.com" />
+	<target host="www.casual-effects.com" />
```

Comment:
```xml
	<target host="cgpp-net.casual-effects.com" />
	<target host="www.cgpp-net.casual-effects.com" />
	<target host="codeheartjs.casual-effects.com" />
	<target host="endlyss.casual-effects.com" />
	<target host="graphicscodex.casual-effects.com" />
	<target host="rocketgolfing.casual-effects.com" />
```

Reply:
My mistake; I forgot to run `sublist3r` for this domain.